addon/content/modules/conversation.js

```diff
@@ -183,7 +183,33 @@ ViewWrapper.prototype = {
   },
 };
 
+// This is a workaround whilst we still have stub.xhtml being loaded in the
+// privileged scope. _ConversationUtils.getBrowser() simulates APIs and passes
+// them back to the webExtension process for handling by the real APIs.
+const SUPPORTED_BASE_APIS = {
+  convContacts: ["beginNew"],
+  conversations: ["createTab"],
+  tabs: ["create"],
+};
+
 class _ConversationUtils {
+  setBrowserListener(listener) {
+    this._browserListener = listener;
+  }
+
+  getBrowser() {
+    const browser = {};
+    for (const [api, subApis] of Object.entries(SUPPORTED_BASE_APIS)) {
```

Comment:
A fancy alternative would be to use a `Proxy` for this. That way you could capture the attribute access and wouldn't need the `SUPPORTED_BASE_APIS`.

Reply:
Thanks, I've used one proxy so far, which gets around having the list of sub apis. Not quite sure it is worth going the double-stacking, but at least for now, this should be a bit easier.